### ISSUE TITLE
Add settings to configure `ansible-playbook --syntax-check`

### ISSUE THREAD

**Issue by cedsana** [ansible/vscode-ansible#520]
### Problem

I have the following setup:
- `ansible`
    - `roles`
    - `dev`
        - `roles`
        - `dev.yml`
    - `ansible.cfg`

Whenever I execute the `dev/dev.yml` playbook, I'm always located inside the `ansible` folder.  
Inside my `ansible.cfg`, I have extended the `roles_path` to tell ansible to look for roles inside the `./roles` as well as the `./dev/roles`.  
My `dev/dev.yml` playbook uses roles from both folders and it works perfectly fine.  

However, when I try to edit the `dev/dev.yml` file in VSCode, I get an error stating that the roles cannot be found:

```
Command failed: ansible-playbook /myrepo/ansible/dev/dev.yml --syntax-check
[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that
the implicit localhost does not match 'all'
ERROR! the role 'myrole' was not found in <default paths>
```

### Solution

From what I found by searching this repository, the README and the settings of the extension, there is no way to configure the behavior of syntax checking when it's done using `ansible-playbook`.
I would like to either disable the checking entirely or be able to change the configuration to tell it to take my `ansible.cfg` into account. Or simply extend the `role_paths` directly even if it's not DRY.

### Alternatives

Disabling the Ansible extension entirely.  
Attempted to switch to `ansible-lint` but ticking the checkbox didn't change anything.

**Comment by priyamsahoo** [comment]
Hi @cedsana, thanks for raising the issue.
One thing, I need to ask. What log (error message) do you get after enabling the ansible-lint option for the extension?

**Comment by cedsana** [comment]
Hi and thanks for answering.

It seems I don't have `ansible-lint` installed:

```
Error: Command failed: which ansible-lint

    at ChildProcess.exithandler (node:child_process:406:12)
    at ChildProcess.emit (node:events:390:28)
    at maybeClose (node:internal/child_process:1064:16)
    at Socket.<anonymous> (node:internal/child_process:450:11)
    at Socket.emit (node:events:390:28)
    at Pipe.<anonymous> (node:net:687:12) {
  killed: false,
  code: 1,
  signal: null,
  cmd: 'which ansible-lint',
  stdout: '',
  stderr: ''
}
```

And right after this error, I get the usual error from `ansible-playbook --syntax-check`.

**Comment by cedsana** [comment]
Yes, opening the `ansible` folder as the vscode root works properly.

**Comment by cedsana** [comment]
That would be a good idea but it wouldn't solve my issue.  
I still think adding a setting for the user to set the `ansible.cfg` would be a good idea.  
I don't want to use the environment variable because this is only an issue on this specific project.  
And I can't move the `ansible.cfg` file to the root of the repository because it would add clutter for the other team members.

Could you keep this issue open, please?
